NEWS.md

```diff
@@ -13,6 +13,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 * fix `default_basis` for `LieGroup` to return a `DefaultLieAlgebraOrthogonalBasis` also when providing a point type. That way `get_vector` falls back to the manifold when called with a Lie group and a point, though this is mere a historical format and the Lie algebra approach is the recommended one.
 * mention `get_coordinates`, `get_vector`, `hat`, and `vee` in the transition documentation since it moved to using the `LieAlgebra` instead of the Lie group and a point.
 * Fixed `RightGroupOperationAction` to be a subtype of `AbstractRightGroupActionType`
+* For the CircleGroup(ℝ), fixed compose StackOverflowError and a bug where result could be outside [-π,π), see #62 for detail.
```

Comment:
```suggestion
* For the CircleGroup(ℝ), fixed compose StackOverflowError and a bug where result could be outside [-π,π), see (#62) for detail.
```

writing issues/PRs is a very good idea. I even stole an idea from JuMP that in the docs these are renedered as links as well (cf https://github.com/JuliaManifolds/LieGroups.jl/blob/fda6db273d1a5c9fdc01b59456fcfd1b7e7cfb49/docs/make.jl#L107-L111) – if they are in round brackets.